### PR TITLE
[MRG] Ch2 technical reviews

### PR DESCRIPTION
**Pull request by hdashnow**

### CODE REVIEW

**Comment by hdashnow** [comment]
This is ready to go :)

File: markdown/ch1.markdown [hunk 1]
```diff
@@ -719,6 +719,45 @@ def binned_boxplot(x, y,
...
+Since version 3.0 Python has enabled us to create
+["keyword-only" arguments](https://www.python.org/dev/peps/pep-3102/).
+These are arguments that you have to call using a keyword, you can't rely on position alone.
+For example, in the `binned_boxplot` function we just wrote, you could call it like this:
+```
+binned_boxplot(x, y, xlabel='my x label', ylabel='my y label')
+```
+but not like this:
+```
+binned_boxplot(x, y, 'my x label', 'my y label')
```
Comment (stefanv) [inline markdown/ch1.markdown]: ... which is possible in Python 2.  (some readers may never have seen this kind of usage before!)

File: markdown/ch2.markdown [hunk 2]
```diff
@@ -80,6 +72,14 @@ As in Chapter 1, first we will use Pandas to make our job of reading in the data
 First we will read in our counts data as a pandas table.
 
 ```python
+# Make all plots appear inline in the Jupyter notebook and set splotting style
+%matplotlib inline
+
+import matplotlib.pyplot as plt
+plt.style.use('style/elegant.mplstyle')
```
Comment (stefanv) [inline markdown/ch2.markdown]: Do we want to filter out these lines, or somewhere in the book show readers where to find it?

Comment (hdashnow) [inline markdown/ch2.markdown]: We can use notedown --precode to hide it. But might be nice to explain in the first chapter. I'll pop it in

**Pull request merged**